mod_reforged/hooks/entity/tactical/actor.nut

```diff
@@ -291,5 +296,29 @@
 					bro.addXP(XPgroup);
 			}
 		}
+
+		this.m.RF_CanDropLoot = ::isKindOf(this, "player") || playerRelevantDamage / this.m.RF_DamageReceived.Total >= 0.5;
+
+		__original(_killer, _skill, _tile, _fatalityType);
 	}
 });
+
+::Reforged.QueueBucket.Late.push(function() {
+	::Reforged.HooksMod.hookTree("scripts/entity/tactical/actor", function(q) {
+		q.getLootForTile = @(__original) function( _killer, _loot )
+		{
+			// Change _killer to null to guarantee loot drop as vanilla checks for killer being null or belonging to Player or PlayerAnimals faction.
+			// Warning: this may be problematic for a mod that hooks this function and expects _killer to be the actual killer.
+			// Note: We do this in Late bucket so that mods that queue their hookTree after Reforged still receive the null _killer.
+			return this.m.RF_CanDropLoot ? __original(null, _loot) : [];
```

Comment:
This method deletes loot which was otherwise meant to always drop, no matter what.
A theoretical example is the **Kraken**, although its items are currently dropped directly during the `onDeath` function